canvas_modules/common-canvas/src/common-properties/form/StructureInfo.js

```diff
@@ -108,7 +114,8 @@ export class StructureDef {
 				propertyOf(uihints)("include_all_fields"),
 				propertyOf(uihints)("layout"),
 				ButtonMetadata.makeButtonMetadata(propertyOf(uihints)("buttons")),
-				propertyOf(structure)("type")
+				propertyOf(structure)("type"),
+				propertyOf(uihints)("disableEditButton")
```

Comment:
Do we need to add it here?  This means it would be included in the paramDef which I'm not seeing a need for.